DataFormats/ParticleFlowReco/src/classes_def_2.xml

```diff
@@ -11,7 +11,6 @@
    <version ClassVersion="11" checksum="1793013291"/>
    <version ClassVersion="10" checksum="1793013291"/>
     <field name="posrep_" transient="true"/>
-    <field name="color_" transient="true"/>
```

Comment:
Rethinhing at it (somehow late...): shouldn't a new class version be defined, since the interface has changed?
@fwyzard

Reply:
I thought about that, run `edmCheckClassVersion`, and nothing came up.
I guess it's because the transient or static members do not count ?

Reply:
Indeed: "Should the class be modified by adding, removing, renaming, or changing the type of any non-transient non-static data member, the new classes_def.xml file might look like [...]" (from https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning)
Thank you for having checked, Andrea